doc/examples/kfold_xval.py

```diff
@@ -97,11 +97,16 @@
 
 fig, ax = plt.subplots(1, 2)
 fig.set_size_inches([12, 6])
-ax[0].plot(cc_vox[~gtab.b0s_mask], dti_cc[~gtab.b0s_mask], 'o', color='b')
-ax[0].plot(cc_vox[~gtab.b0s_mask], csd_cc[~gtab.b0s_mask], 'o', color='r')
-ax[1].plot(cso_vox[~gtab.b0s_mask], dti_cso[~gtab.b0s_mask], 'o', color='b', label='DTI')
-ax[1].plot(cso_vox[~gtab.b0s_mask], csd_cso[~gtab.b0s_mask], 'o', color='r', label='CSD')
-plt.legend(loc='upper left')
+ax[0].plot(cc_vox[~gtab.b0s_mask], dti_cc[~gtab.b0s_mask], 'o', color='b',
```

Comment:
Can we get rid of `~` and use `not`?  I know it is not related to your PR but many people do not understand this syntax, This is a recurrent question.  Let me know if you want to update it otherwise, it is ready to go, I can merge it when the CI's are happy

Reply:
I'd be fine with that.

Reply:
@skoudoro and @arokem  thanks for pointing this out.. will fix it :)

Reply:
Done! Looks good?